File: 22y 10m/1011/MyHelloJava05/src/javastudy02/Animal.java
package javastudy02;

public abstract class Animal {
	private String Name;
	private int Age;
	public abstract void sleep();
	public String getName() {
		return Name;
	}
	public void setName(String name) {
		Name = name;
	}
	public int getAge() {
		return Age;
	}
	public void setAge(int age) {
		Age = age;
	}
	
}

File: 22y_11m/1117/HelloCSharp07/HelloCSharp07_03/Animal.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HelloCSharp07_03
{
    abstract public class Animal
    {
        public string Name { get; set; }
        public int Age { get; set; }
        public void sleep()
        {
            System.Windows.Forms.MessageBox.Show($"{Name}은 {Age}살이라서 많이 잡니다.");
        }
        public abstract void eat();
    }
}
